.gitignore

```diff
@@ -23,4 +23,17 @@ profiles.clj
 .idea
 resources/contracts
 node_modules
+
+# OS generated files #
+######################
```

Comment:
How about managing this with a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)? :)